crates/store/re_types/definitions/rerun/components/fill_mode.fbs

```diff
@@ -4,13 +4,18 @@ namespace rerun.components;
 enum FillMode: byte(
```

Comment:
Not new in this PR, but the downside with the `FillMode` design is that you cannot have both a wireframe and a fill color. Once we have transparency implemented, I think that is something users will want to have.

For example: I want to show a translucent sphere, but with the three axis circles on it opaque.

Reply:
Yep. The entirety of `FillMode` is a temporary workaround for lack of a way to set the colors independently, as I see it. I should add a (non-documentation) comment to that effect.

Reply:
Done.